scripts/bumpPackages/index.js

```diff
@@ -1,56 +1,49 @@
-const { exec } = require('child_process');
+const { execSync } = require('child_process');
```

Comment:
Ahh I attempted to use this, I tried to get it to work with promises which makes sense why it didn't work. I need to look up more about async and sync in JS.